src/prefect/client/schemas/objects.py

```diff
@@ -501,6 +504,9 @@ def populate_deprecated_fields(cls, values: Any):
         return values
 
 
+KeyValueLabels = Dict[str, Union[StrictBool, StrictInt, StrictFloat, str]]
```

Comment:
🧐 I thought this was already defined in this file? https://github.com/PrefectHQ/prefect/blob/main/src/prefect/client/schemas/objects.py#L74